spylib/token.py

```diff
@@ -265,6 +276,76 @@ async def execute_gql(
 
         return jsondata['data']
 
+    async def create_http_webhook(
+        self,
+        topic: Union[WebhookTopic, str],
+        callback_url: str,
+        include_fields=None,
+        metafield_namespaces=None,
+        private_metafield_namespaces=None,
+    ) -> WebhookResponse:
+        """Uses graphql to create a webhook"""
+        query = '''
+        mutation webhookSubscriptionCreate($topic: WebhookSubscriptionTopic!,
+                                           $webhookSubscription: WebhookSubscriptionInput!) {
+            webhookSubscriptionCreate(topic: $topic, webhookSubscription: $webhookSubscription) {
+                webhookSubscription {
+                id
+                topic
+                format
+                endpoint {
+                    __typename
+                    ... on WebhookHttpEndpoint {
+                    callbackUrl
+                    }
+                  }
+                }
+                userErrors {
+                    field
+                    message
+                }
+              }
+            }
+        '''
+        variables = {
+            'topic': topic.value if isinstance(topic, WebhookTopic) else topic,
+            'webhookSubscription': {
+                'callbackUrl': callback_url,
+                'format': WEBHOOK_JSON_FORMAT,
+                'includeFields': include_fields,
+                'metafieldNamespaces': metafield_namespaces,
+                'privateMetafieldNamespaces': private_metafield_namespaces,
+            },
+        }
+        res = await self.execute_gql(query=query, variables=variables)
+        webhook_create = res.get('webhookSubscriptionCreate', None)
+        if webhook_create and webhook_create.get('userErrors', None):
+            raise ShopifyGQLUserError(res)
+        return WebhookResponse(id=webhook_create['webhookSubscription']['id'])
+
+    async def create_event_bridge_webhook(
+        self,
+        topic: Union[WebhookTopic, str],
+        arn: str,
+        include_fields=None,
+        metafield_namespaces=None,
+        private_metafield_namespaces=None,
+    ) -> WebhookResponse:
+        # TODO
+        pass
```

Comment:
I think that'd be clearer:
```suggestion
        raise NotImplementedError
```